RevenueCatUI/Templates/V2/PaywallsV2View.swift

```diff
@@ -237,23 +237,47 @@ private struct LoadedPaywallsV2View: View {
     }
 
     var body: some View {
-        VStack(spacing: 0) {
-            ComponentsView(
-                componentViewModels: [paywallState.componentViewModel],
-                onDismiss: self.onDismiss
+        GeometryReader { proxy in
+            VStack(spacing: 0) {
+                ComponentsView(
+                    componentViewModels: [.root(paywallState.rootViewModel)],
+                    onDismiss: self.onDismiss
+                )
+            }
+            // If the first view in the first stack is an image,
+            // we will ignore safe area pass the safe area insets in to environment
+            // If the image is in a ZStack, the ZStack will push non-images
+            // down with the inset
```

Comment:
I'm also wondering if we should make it so we only apply this if the image width is `Fill`, so it uses the whole space... In case there is something like this:
<img width="385" alt="image" src="https://github.com/user-attachments/assets/7158e3c3-ad89-44b7-b528-eb547a2f402b" />

Reply:
Oooops, you are right! Its supposed to only remove safe area inset if hte image has a width of `fill` 🤦‍♂️ WIll fix!